### DISCUSSION TITLE
Unique number between 1000 and 2000 ?

### DISCUSSION THREAD

**Discussion by FVolral** [faker-js/faker#1169] [Q&A]
Can I do this? 

```
faker.seed(4242)
const id = faker.unique(faker.datatype.number, { min: 1000, max: 2000, precision: 0.01 })
```

Here `id` equals 32494. Why the methods args aren't applied ?

**Comment by Shinigami92** [comment]
Try `faker.unique(faker.datatype.number, [{ min: 1000, max: 2000, precision: 0.01 }])`
